### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by ueman**
## :scroll: Description

This PR
- fixes a couple of broken code references
- improves formatting
- adds recently added packages to the ignored frames

## :green_heart: How did you test it?

No functionality changes, so existing tests cover these changes.

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] I added tests to verify changes
- [x] No new PII added or SDK only sends newly added PII if `sendDefaultPii` is enabled
- [x] I updated the docs if needed
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: flutter/example/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -460,7 +460,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (ueman) [inline flutter/example/ios/Runner.xcodeproj/project.pbxproj]:
```suggestion
				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

File: flutter/example/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -631,7 +631,7 @@
...
```
Comment (ueman) [inline flutter/example/ios/Runner.xcodeproj/project.pbxproj]:
```suggestion
				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

File: flutter/example/ios/Runner.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -681,7 +681,7 @@
...
```
Comment (ueman) [inline flutter/example/ios/Runner.xcodeproj/project.pbxproj]:
```suggestion
				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

File: flutter/lib/src/sentry_flutter_options.dart [hunk 4]
```diff
@@ -8,6 +8,9 @@ import 'binding_wrapper.dart';
 import 'renderer/renderer.dart';
 import 'screenshot/sentry_screenshot_quality.dart';
 import 'event_processor/screenshot_event_processor.dart';
+import 'screenshot/sentry_screenshot_widget.dart';
+import 'sentry_flutter.dart';
+import 'user_interaction/sentry_user_interaction_widget.dart';
```
Comment (ueman) [inline flutter/lib/src/sentry_flutter_options.dart:13]: This fixes the code references further down in the file.

**Review by buenaflor — APPROVED** [review]
thx for the cleanup! I'll see to it that the tests are green

**Comment by stefanosiano** [comment]
hey @ueman 
thanks for the cleanup! Can we merge it or do you want to add something else?

**Review by stefanosiano — APPROVED** [review]

**Comment by ueman** [comment]
It can be merged, I don't have anything to add

**Pull request merged**